### PR TITLE
Invert disableAutoTool to autoTool

### LINKED ISSUE #2632: Rename the `disableAutoTool` setting to `autoTool` and invert the value, to be more intuitive to users
## Describe your suggestion
Rename `disableAutoTool` to `autoTool` and invert the value.

## Settings
`#autoTool true // default value`
`#autoTool false // disabled autotool`

## Context
This would be more intuitive to users.

### PR DESCRIPTION
**Pull request by ZacSharp**
closes #2632

### CODE REVIEW

**Review by l1ving — APPROVED** [review]
lol thank you

**Review by scorbett123 — APPROVED** [review]

File: src/api/java/baritone/api/Settings.java [hunk 1]
```diff
@@ -72,7 +72,7 @@
     /**
      * If this setting is on, no auto tool will occur at all, not at calculation time nor execution time
```
Comment (Fongeme) [inline src/api/java/baritone/api/Settings.java:73]: Shouldnt you update documentation aswell?

Comment (Fongeme) [inline src/api/java/baritone/api/Settings.java:73]:
```suggestion
     * If this setting is off, no auto tool will occur at all, not at calculation time nor execution time
```

Comment (Fongeme) [reply src/api/java/baritone/api/Settings.java:73]: im unsure of any other places where this might change.

Comment (ZacSharp) [reply src/api/java/baritone/api/Settings.java]: Did a grep for 'autoTool' over the whole repo and didn't find any more.
Sorry for not accepting your suggestion, but I wanted to have the shorter wording.

Comment (Fongeme) [reply src/api/java/baritone/api/Settings.java]: LGTM
